src/ds/d500/d500-auto-calibration-handler.cpp

```diff
@@ -0,0 +1,130 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#include "d500-auto-calibration-handler.h"
+
+
+namespace librealsense
+{
+    d500_auto_calibrated_handler_hw_monitor::d500_auto_calibrated_handler_hw_monitor()
+    {
+
+    }
+
+    d500_calibration_answer d500_auto_calibrated_handler_hw_monitor::get_status() const
+    {
+        if (auto hwm = _hw_monitor.lock())
+        {
+            auto res = hwm->send(command{ ds::GET_CALIB_STATUS });
+            // checking size of received buffer
+            if (!check_buffer_size_from_get_calib_status(res))
+                throw std::runtime_error("GET_CALIB_STATUS returned struct with wrong size");
+
+            return *reinterpret_cast<d500_calibration_answer*>(res.data());
+        }
+        throw std::runtime_error("hw monitor has not bewen set");
+    }
+
+    std::vector<uint8_t, std::allocator<uint8_t>> d500_auto_calibrated_handler_hw_monitor::run_auto_calibration(d500_calibration_mode _mode)
+    {
+        if (auto hwm = _hw_monitor.lock())
+        {
+            return hwm->send(command{ ds::SET_CALIB_MODE, static_cast<uint32_t>(_mode), 1 /*always*/ });
```

Comment:
alwayes?
What does it means?

Reply:
means that the input here is always 1 - it means the calibration type is triggered calibration. I can create an enum, but this is the only value, so it means to be needless to me.

Reply:
I would rename the comment from `always` to `1 --> Triggered Calibration`
Since cannot know this hint from the code